File: problem06-fizzBuzz/fizzBuzz.py
'''
@name fizzBuzz 
@param n - the total value to iterate up to 
@desc A program that iterates from 1 to n but if the current value is 
@param n - max integer that iterates up to 
'''
def fizzBuzz(n):
    maxVal = n + 1
    for i in range(1, maxVal):
        if (i % 3 == 0 and i % 5 != 0):
            print("fizz") 
        elif (i % 5 == 0 and i %3 != 0):
            print("buzz")
        elif (i % 3 == 0 and i % 5 ==0): 
            print("fizzbuzz")
        else:
            print(i)
fizzBuzz(12)

File: problem06-fizzBuzz/fizzBuzz.js
/**
 * @name fizzBuzz
 * @desc A program that console logs the number *n* but if n is a multiple of 3 print fizz, else if 5 print buzz, if both print fizzbuzz.
 * @param n - integer
 */
function fizzBuzz(n) {
  for (let i = 1; i < n + 1; i++) {
    if (i % 3 === 0 && i % 5 !== 0) {
      console.log('fizz');
    } else if (i % 5 === 0 && i % 3 !== 0) {
      console.log('buzz');
    } else if (i % 5 === 0 && i % 3 === 0) {
      console.log('fizzbuzz');
    } else {
      console.log(i);
    }
  }
}
